units/Watched_Pot_Drone.cfg

```diff
@@ -0,0 +1,128 @@
+#textdomain wesnoth-loti
+[unit_type]
```

Comment:
Can you use `base_unit`?

Reply:
I think I made a change to just about everything in the drones, so there's nothing really left to inherit from what I understand of base_unit.

They do need descriptions, and this one could probably use a name (I couldn't think of anything that sounds like a unit that makes things slow).

Reply:
The animation code is still duplicated.

Reply:
the sound is changed, so the only thing that would be inherited is a few lines of standing animation.